Makefile

```diff
@@ -1,3 +1,5 @@
+KIND_CLUSTER=kind
```

Comment:
`kind` is the default one, so for better isolation, let's use a specific name, e.g. `agent-sandbox`